content/blog/2021/07/camel-k-release-1-5/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Camel K 1.5 release"
+date: 2021-07-13
+draft: false
+authors: [squakez]
+categories: ["Releases", "Camel K"]
+preview: "What's new in Camel K 1.5"
+---
+
+[Camel K 1.5 version](https://github.com/apache/camel-k/releases/tag/v1.5.0) is public since a few days. We're proud to announce this new release containing enhancements and fixes. We've worked hard to fix the most important issues discovered with an eye on stabilization and performances. This new release is based on **Camel Quarkus 2.0** and **Apache Camel 3.11**.
+
+Let's find out what's new in Camel K 1.5!
+
+## New Camel Quarkus and Quarkus versions
+
+**Camel Quarkus** runtime has become the building block on which the `Integration` is materialized. With the new 2.0 version out, there are great improvements which are leveraging the major release of **Quarkus 2.0**. We're inheriting all the good stuff in performance and components coverage that is introduced lately in this new version.
+
+* [Camel Quarkus 2.0.0 released](/blog/2021/06/camel-quarkus-release-2.0.0/)
+* [What's new in Camel 3.11.0](/blog/2021/06/Camel311-Whatsnew/)
+
+## Tune your logs
+
+As software developers, we are daily watching logs to discover what's going on for a piece of software that is failing. **Observability** and all the activities around that area are needed in order to have a reliable software solution. We know that, and we've introduced a new `trait` that will simplify the work of tuning and integrating the log streams in your logging systems.
+
+You can learn more in [a blog dedicated to the topic](/blog/2021/05/new-camel-k-logging-features/).
+
+## S3 dependencies
+
+**S3** protocol is often used as a way to store Maven dependencies. With version 1.5 you will be able to configure your Camel K to retrieve such dependencies.
```

Comment:
What about?

```suggestion
## Amazon S3 dependencies

**Amazon S3** protocol is often used as a way to store Maven dependencies. With version 1.5 you will be able to configure your Camel K to retrieve such dependencies.
```

Reply:
Although it was originally provided by Amazon, I think S3 can be considered as an agnostic protocol. I mean, you can use it with other services such as Ceph.

Reply:
(i do not have rights to resolve my own comments)